content/applications/sales/crm/acquire_leads/convert.rst

```diff
@@ -34,24 +34,26 @@ done, click :guilabel:`Save`.
 Convert a lead into an opportunity
 ==================================
 
-To convert a lead into an *opportunity*, navigate to :menuselection:`CRM --> Leads`, and click on a
-:guilabel:`Lead` from the list to open it.
+To convert a lead into an *opportunity*, navigate to :menuselection:`CRM app --> Leads`, and click
+on a lead from the list to open it.
 
-In the upper-left corner of the screen, click the :guilabel:`Convert to Opportunity` button, which
-opens a :guilabel:`Convert to opportunity` pop-up window.
+.. warning::
+   If a :guilabel:`Similar Leads` smart button appears at the top of the page for the lead, it
+   indicates a similar lead or opportunity already exists in the database. Before converting this
+   lead, click the smart button to confirm if the lead should be merged.
+
+   .. image:: convert/similar-leads-smart-button.png
+      :align: center
+      :alt: Close up of a lead with emphasis on the Similar Leads smart button.
+
+Click the :guilabel:`Convert to Opportunity` button, located at the top-left of the page.
 
 .. image:: convert/convert-leads-convert-opp-button.png
    :align: center
    :alt: Create opportunity button on a lead record.
 
-On the :guilabel:`Convert to opportunity` pop-up window, in the :guilabel:`Conversion action` field,
-select the :guilabel:`Convert to opportunity` option.
-
-.. note::
-   If a lead, or an opportunity, already exists in the database for this customer, Odoo
-   automatically suggests merging both leads/opportunities. For more information on merging leads
-   and opportunities, see the section on how to :ref:`merge leads
-   <sales/crm/acquire_leads/merge-leads>` below.
+This opens a :guilabel:`Convert to opportunity` pop-up modal. Here, in the :guilabel:`Conversion
```

Comment:
I used "modal" per Sam's instructions (he has been updating this in tech reviews so I decided to save a step). We are going to try to nail down the exact usage for each of these terms (popover, pop-up and modal) in the near future, but for the time being, I have been using "Inspect" to identify what element is being used.
I removed the mention of the merge option for now only because I want to be able to link to the new doc that is currently in the process of being added (so I will have to make that change once the doc is published) but you are right, I should make sure that option is mentioned :+1:

Reply:
@jero-odoo fully aware of the ongoing convo re: "modal" v. "pop-ups" -- the most *recent* development is that we should try to lean towards a more novice-friendly approach with these terms -- at least for the time being (per ZST's most recent "ruling" on the matter)... but, again, I fully understand and appreciate the logic behind using the technical/accurate term. However, it just seems like that may be a little too "inside baseball" for users who aren't super versed in the intricacies of UI terminology. That said, ZST, SALI, and I have all agreed to take a more "approachable" route with these terms -- for now -- and then we'll *all* discuss our thoughts on the matter in an upcoming Writer's Meeting.